var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -83,5 +87,51 @@ def configure_args(self):
 
         args.append('--with-scheduler=%s'
                     % self.spec.variants['scheduler'].value)
-
         return args
+
+    @run_after('install')
+    def setup_build_tests(self):
+        """Copy the build test files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        tests = self.test_list
+        relative_test_dir = self.test_base_path
+        files_to_cpy = []
+        header = 'test/argparsing.h'
+        for test in tests:
+            test_path = join_path(relative_test_dir, test + '.c')
+            files_to_cpy.append(test_path)
+        files_to_cpy.append(header)
+        self.cache_extra_test_sources(files_to_cpy)
+
+    def build_tests(self):
+        """Build and run the added smoke (install) test."""
+        tests = self.test_list
+        relative_test_dir = self.test_base_path
+
+        for test in tests:
+            options = [
+                      '-I{0}'.format(self.prefix.include),
```

Comment:
Python is picky.  Lines 113 to 121 should be indented 4 spaces relative to line 112, with two exceptions.

The first letter of `test` in line 116 should appear under the `s` in line 115.  A similar alignment is needed on line 122.